ci-operator/config/openshift/cluster-baremetal-operator/openshift-cluster-baremetal-operator-release-4.10.yaml

```diff
@@ -106,6 +106,26 @@ tests:
   steps:
     cluster_profile: packet
     workflow: baremetalds-e2e-serial-ipv4
+- as: e2e-metal-ipi-serial-virtualmedia
+  skip_if_only_changed: (^[A-Z]+\.md$)|((^|/)OWNERS$)
```

Comment:
you may want to add LICENSE and PROJECT
here and to all the other regexes

Reply:
Thanks @elfosardo. These jobs are added to test the serial workflows are working, thus they are not part of this PR. However, after this is merged, eventually new jobs will be added and I'll add LICENSE an PROJECT and use other regexes as you suggested.

Reply:
The LICENSE never changes so it's academic.
PROJECT is a config file for kubebuilder... I guess that doesn't run at build time so it might be excludable in theory, but in practice it never changes without some code also changing.
In short, this is not worth worrying about.